docs/guides/modules/test/pages/smarter-testing.adoc

```diff
@@ -41,565 +41,1571 @@ TIP: In codebases with sparse test coverage, Smarter Testing cannot accurately d
++
+Verify with:
++
+[source,console]
+----
+$ sha256sum -c <(echo "a5770f0c964b434df3560358836fa2401f526379e84aac2cd6c7946180ed1ce6  ./circleci-testsuite.gz")
+----
 
-[source,yaml]
+Windows AMD64::
+https://circleci-binary-releases.s3.amazonaws.com/circleci-cli-plugins/circleci-testsuite/1.0.15108-d839eb2/windows/amd64/circleci-testsuite.exe.gz[Download]
++
+Verify with:
++
+[source,console]
 ----
-# .circleci/test-suites.yml
-name: ci tests
-discover: jest --listTests --testPathPattern=src/
-run: ##
-outputs:
-  junit: ##
+$ sha256sum -c <(echo "ca0f395b1ffeca557428a30cc405ad193fe681792822cfae73c7c7c813687c0c  ./circleci-testsuite.exe.gz")
 ----
+====
 
-*Examples for the `discover` command*
+After downloading the plugin and verifying the checksum:
 
-[.table-scroll]
---
-[cols=2*, options="header"]
-|===
-| Test Framework| Command
+. Decompress the binary
+. Set the execute bit
+. Move it to a directory on your shell's `$PATH`
 
-|*Jest*
-|`jest --listTests`
+[source,console]
+----
+$ gunzip circleci-testsuite.gz
+$ chmod u+x circleci-testsuite
+$ mv circleci-testsuite "${HOME}/bin/"
+----
 
-|*Yarn Jest*
-|`yarn --silent test --listTests --testPathPattern=src/`
 
-|*Vitest*
-|`vitest list --filesOnly`
+=== 1.2 Configure the discover command
+TIP: The `circleci run testsuite` CLI tooling must be run from your repository root directory. The commands run as part of the test suite will also be run from your repository root directory, and should not use `cd` to change the working directory.
 
-|*pytest*
-|`pytest --collect-only -qq \| sed 's/:.*//' \| sort -u`
+Smarter Testing operates on "test atoms".
 
-|*Go*
-|`++go list -f '{{ if or (len .TestGoFiles) (len .XTestGoFiles) }} {{ .ImportPath }} {{end}}' ./...++`
+A "test atom" is a single runnable unit that exercises one or more tests, depending on the specific test runner being used. for example `go test pkg/foo`, `jest index.test.ts`, `./gradlew test --tests "TestMethod"`
 
-|===
---
+The `discover` command finds all test atoms for a given test suite. The stdout of this command is split on both newlines and whitespace, each token is treated as a test atom.
 
-=== 1.4 Populate the run command
+The `discover` command should not execute any tests.
 
-The `run` command executes the tests discovered by the `discover` command using a test runner.
+Every line of stdout will be interpreted as test atoms, any tool that prints metadata in the stdout will need to be suppressed.
 
-"Discovered" tests can be run in one of two ways:
+It may be useful to run the `discover` command in your shell to examine the output.
 
-* Use the template variable `<< test.atoms >>` in the `run` command. This will be replaced with a space-separated list of tests to run.
-* If the template variable is not found in the `run` command, each test will be newline-separated in stdin.
+.Create a `.circleci/test-suites.yml` file in the project root and populate the `discover` command
```

Comment:
It feels like there’s a missing transition between the explanation of what a discovery is and the steps for configuring the discovery command. Adding a brief connector would help the flow.